### DISCUSSION TITLE
Problem with valid images not being tagged

### DISCUSSION THREAD

**Discussion by gomsis** [ultralytics/yolov5#7759] [Q&A]
I have been training my own model using yolov5, but I am having trouble learning it.
When we used our own dataset and had it trained, we got these results.
![results](https://user-images.githubusercontent.com/92967049/167778679-4fd74556-4418-499c-847b-4b2bd87e5785.png)
In the above learning, for some reason it stopped in the middle of the study, so we reduced the number of epochs and ran the study again.
Here are the results.
![results](https://user-images.githubusercontent.com/92967049/167779631-2b72b2f2-83d2-44e3-800e-3aeabc526976.png)
This is also a situation where mAP drops sharply and for some reason learning is not going well.

Also, the output train_batch0_pred.jpg, train_batch1_pred.jpg, etc. are not tagged at all.
I thought my homebrew dataset was causing the problem, so I attempted to train with the coco128 dataset.
However, the results with coco128 are similar and I suspect that the environment is the problem
![results](https://user-images.githubusercontent.com/92967049/167779893-010c8bd9-d398-45c2-b760-9f95f4f20aa3.png)
.
I am having trouble finding the cause.
Since I could not find it by searching in Japanese, I am translating it and posting it in English. Please help me even though my English is not very good.

**Comment by glenn-jocher** [comment]
@gomsis you have no val labels or your val labels are incorrect. Without val labels there will obviously not be any metrics.

**Reply by gomsis** [reply]
Thank you so much for all your answers!
However, we are wondering if there is some other cause since we are not getting the same metrics using coco128, which is a prepared data set.
I also do not understand why presicion and recall drop so rapidly in my own data set.

I look forward to working with you

**Reply by glenn-jocher** [reply]
@gomsis you have no val labels or your val labels are incorrect. Without val labels there will obviously not be any metrics.
